mne/viz/evoked.py

```diff
@@ -1191,3 +1191,342 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
+        keys are used as condition labels.
+        If a [dict/list] of lists, the unweighted mean is plotted as a time
+        series and the parametric confidence interval is plotted as a shaded
+        area. All instances must have the same shape - channel numbers, time
+        points etc.
+    picks : int | list of int | str | list of str | None
+        The indices (int) or names (str) of the sensors to plot. Must all
+        have the same channel type.
+        If the selected channels are gradiometers, the corresponding pairs
+        will be selected.
+        If None, the global field power is plotted instead.
+    conditions : list of str | None
+        If evokeds is a single instance or a list (but not a dictionary),
+        this can be used for legend labels.
+    colors : list | dict | None
+        If a list, will be sequentially used for line colors.
+        If a dict, can map evoked keys or HED-like tags to conditions. For
+        example, if `evokeds` is a dict with the keys "Aud/L", "Aud/R",
+        "Vis/L", "Vis/R", `colors` can be dict(Aud='r', Vis='b') to map both
+        Aud/L and Aud/R to the color red.
+        If None, a sequence of desaturated colors is used.
+    linestyles : list | dict
+        If a list, will be sequentially used for evoked plot linestyles.
+        If a dict, can map the `evoked` keys or HED-like tags to conditions.
+        For example, if evokeds is a dict with the keys "Aud/L", "Aud/R",
+        "Vis/L", "Vis/R", `linestyles` can be dict(L='--', R='-') to map both
+        Aud/L and Vis/L to dashed lines.
+    styles : dict | None
+        If a dict, keys must map to evoked keys or conditions, and values must
+        be a dict of legal inputs to `matplotlib.pyplot.plot`. These
+        parameters will be passed to the line plot call of the corresponding
+        condition, overriding defaults.
+    vlines : list
+        A list of integers corresponding to the positions, in milliseconds,
+        at which to plot dashed vertical lines.
+    ci : float | None
+        If not None and `evokeds` is a [list/dict] of lists, a confidence
+        interval is drawn around the individual time series. This value
+        determines the CI width. E.g., if this value is .95 (the default),
+        the 95% parametric confidence interval is drawn.
+        If None, no shaded error band is plotted.
+    truncate_yaxis : bool
+        If True, the left y axis is truncated to reduce visual clutter.
+    invert_y : bool
+        Should negative values be plotted up (as is sometimes done for ERPs
+        out of tradition)? Defaults to False.
+    ax : None | `matplotlib.pyplot.axes` instance
+        What axes to plot to. If None, a new axes is created.
+    ymin, ymax : float | None
+        If not `None`, can be used to manually scale the y axis.
+    title : None | str
+        If str, will be plotted as figure title. If None, the channel
+        names will be shown.
```

Comment:
if passing ymin and ymax changes how y axis is truncated than ok.
title seems unnecessary to me because one can easily do `fig.axes[0].set_title('some title')` but I'm ok with leaving it.